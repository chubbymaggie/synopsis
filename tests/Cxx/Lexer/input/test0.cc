// a comment

int i;

/*...
 */

struct foo { double bar;};
