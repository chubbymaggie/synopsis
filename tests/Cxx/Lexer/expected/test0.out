292    : "int"
258    : "i"
 59 ';': ";"
303    : "struct"
258    : "foo"
123 '{': "{"
286    : "double"
258    : "bar"
 59 ';': ";"
125 '}': "}"
 59 ';': ";"
